accounting/accounting.go

```diff
@@ -243,8 +258,9 @@ func (accounting *AccountingState) AddTransaction(txnr *idb.TxnRow) (err error)
 	accounting.updateAlgo(accounting.feeAddr, int64(stxn.Txn.Fee))
 
 	if stxn.SenderRewards != 0 {
-		accounting.updateAlgo(stxn.Txn.Sender, int64(stxn.SenderRewards))
-		accounting.updateAlgo(accounting.rewardAddr, -int64(stxn.SenderRewards))
+		v := int64(stxn.SenderRewards)
```

Comment:
Consider giving the variable full names. It makes it difficult to follow.

Reply:
I normally agree with that, but here I intentionally kept the short variable to make it easier to see that it's being repeated in several places. Your other suggestion to make a function which avoids repeating the value is probably a better solution.